thrift/history.thrift

```diff
@@ -216,6 +217,7 @@ struct RecordDecisionTaskStartedResponse {
   120: optional i64 (js.type = "Long") scheduledTimestamp
   130: optional i64 (js.type = "Long") startedTimestamp
   140: optional map<string, shared.WorkflowQuery> queries
+  150: optional i64 (js.type = "Long") historySize
```

Comment:
should the name be totalhistorybytes?

Reply:
This is the field sent from cadence history to cadence matching, so I was thinking of keeping the same name used in persistence where we calculate the history size.